app/models/profile.rb

```diff
@@ -13,9 +13,6 @@ class Profile < ApplicationRecord
   belongs_to :benchmark, class_name: 'Xccdf::Benchmark'
   belongs_to :parent_profile, class_name: 'Profile', optional: true
 
-  validates :ref_id, uniqueness: {
```

Comment:
Incorrect addition. Uniqueness already handled in master, PR #613.

Reply:
This is the large uniqueness condition that matches the DB constraint. It's not incorrect, just duplicated by #613, right?

Reply:
Yes, and no. It was missing the condition `if: :policy_id`.